globalwarmingpotentials.csv

```diff
@@ -1,98 +1,115 @@
+HFC134,1000,1100,,1120,1337,1260,3200,3900,330,361,239
+HFC134a,1300,1300,1430,1300,1549,1530,3300,4140,400,436,306
+HFC143,300,330,,328,397,364,1100,1300,100,104,66.6
+HFC143a,3800,4300,4470,4800,5508,5810,5500,7840,1600,1940,3250
+HFC152,,43,,16,20,21.5,140,77.6,13,6.14,3.89
+HFC152a,140,120,124,138,167,164,410,591,37,46.8,29.8
+HFC161,,12,,4,4,4.84,40,17.4,4,1.38,0.872
+HFC227ea,2900,3500,3220,3350,3860,3600,5600,5850,1100,1100,1490
+HFC236cb,,1300,,1210,1438,1350,3300,3750,390,387,268
+HFC236ea,,1200,,1330,1596,1500,3600,4420,390,428,288
+HFC236fa,6300,9400,9810,8060,8998,8690,7500,7450,7100,6040,8870
+HFC245ca,560,640,,716,863,787,2100,2680,200,225,146
+HFC245fa,,950,1030,858,1032,962,3000,3170,300,274,180
+HFC365mfc,,890,794,804,966,914,2600,2920,280,261,172
+HFC4310mee,1300,1500,1640,1650,1952,1600,3700,3960,470,458,347
+SO2F2,,,,4090,4732,4630,,7510,,1410,1920
+SF6,23900,22200,22800,23500,26087,25200,15100,18300,32400,34100,30600
+NF3,,10800,17200,16100,17885,17400,7700,13400,13100,18200,20000
+CF4,6500,5700,7390,6630,7349,7380,3900,5300,8900,10600,9050
+C2F6,9200,11900,12200,11100,12340,12400,8000,8940,18000,17500,15200
+C3F8,7000,8600,8830,8900,9878,9290,5900,6770,12400,12400,11200
+cC4F8,8700,10000,10300,9540,10592,10200,6800,7400,14500,13800,12400
+C4F10,7000,8600,8860,9200,10213,10000,5900,7300,12400,13400,12100
+C5F12,7500,8900,9160,8550,9484,9220,6000,6680,13200,12700,11200
+C6F14,7400,9000,9300,7910,8780,8620,6100,6260,13200,11600,10500
+C7F16,,,,7820,8681,8410,,6120,,11300,10200
+C8F18,,,,7620,8456,8260,,6010,,11100,10000
+C10F18,,,,7190,7977,7480,,5480,,9780,9010
+SF5CF3,,,17700,17400,19396,18500,,13900,,21100,21600
+cC3F6,,,,9200,10208,,,,,,
+HFE125,,14900,14900,12400,13951,14300,12900,13500,9200,7680,13100
+HFE134,,6100,6320,5560,6512,6630,10500,12700,2000,1940,2060
+HFE143a,,750,756,523,632,616,2500,2170,230,176,113
+HCFE235da2,,340,350,491,595,539,1100,1930,110,154,98.4
+HFE245cb2,,580,708,654,790,747,1900,2630,180,213,137
+HFE245fa2,,570,659,812,981,878,1900,3060,180,251,162
+HFE254cb2,,30,,,,,99,,9,,
+HFE347mcc3,,,575,530,641,576,1600,2020,150,164,106
+HFE347pcf2,,,580,889,1072,980,,3370,,279,181
+HFE356pcc3,,110,110,413,500,277,,995,,79,50.4
+HFE569sf2,,,59,57,69,60.7,,219,,17.3,11
+HFE4310pccc124,,,1870,2820,3353,3220,,8720,,920,647
+HFE236ca12,,,2800,5350,6260,6060,,11700,,1770,1860
+HFE338pcc13,,,1500,2910,3466,3320,,9180,,948,657
+HFE227ea,,1500,,6450,7377,7520,4200,9800,460,2570,4440
+HFE236ea2,,960,,1790,2143,2590,3100,7020,300,741,521
+HFE236fa,,470,,979,1177,1100,1600,3670,150,315,205
+HFE245fa1,,280,,828,997,934,940,3170,86,266,173
+HFE263fb2,,11,,1,2,,37,,3,,
+HFE329mcc2,,890,,3070,3598,3770,2800,7550,280,1100,1090
+HFE338mcf2,,540,,929,1118,1040,1800,3460,170,297,194
+HFE347mcf2,,360,,854,1028,963,1200,3270,110,275,179
+HFE356mec3,,98,,387,468,264,330,949,30,75.3,48
+HFE356pcf2,,260,,719,867,831,860,2870,80,237,154
+HFE356pcf3,,430,,446,540,484,1500,1730,130,138,88.4
```

Comment:
TAR has HFE356pcc3, other reports have HFE356pcf3. Not sure if one is a typo or whether they're actually different.

Reply:
This document https://faolex.fao.org/docs/pdf/mex148237.pdf has

```
HFE356pcc3; 1,1,2,2,3,3-Hexafluoropropil metil éter; (CH3OCF2CF2CHF2): CAS 160620202
```

and
```
HFE356pcf3; 2,2,3,3-Tetrafluoropropil difluorometil éter; (CHF2OCH2CF2CHF2): CAS
35042990
```

In general my feeling would be to include the data as published in the original IPCC sources, unless it's an obvious typo.

Reply:
As I said above, both exist, we have values for both, they are distinct.

Reply:
Even better, @mikapfl :-)